### PR TITLE
Transaction process 2/3: Read from new schema

### PR DESCRIPTION
**Pull request by rap1ds**
- [x] Read from TransactionProcess

### CODE REVIEW

**Comment by Gnito** [comment]
:+1:

File: app/controllers/listings_controller.rb [hunk 1]
```diff
@@ -110,15 +110,19 @@ def show
...
-    payment_process = select_payment_process(
-      price_field: @listing.transaction_type.price_field?,
-      preauthorize: @listing.transaction_type.preauthorize_payment?,
-      payment_gateway_available: payment_gateway.present?)
+    # TODO Change this so that the path is always the same, but the controller
+    # decides what to do. We don't want to make a API call to TransactionService
+    # just to show a listing details
+    process_res = TransactionService::API::Api.processes.get(
+      community_id: @current_community.id,
+      process_id: @listing.transaction_type.transaction_process_id
```
Comment (ovan) [inline app/controllers/listings_controller.rb]: Listing must record transaction_process_id reference at create/edit and use that. Is the plan to change that part at later? If so then perhaps add a TODO comment to not miss it later.

Comment (rap1ds) [inline app/controllers/listings_controller.rb]: Yes, that will be done later.

File: app/services/transaction_service/process.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Request: Empty
+
+Response:
+
+
+```json
+{
+  id: 123,
+  process: "preauthorize" # or "none" or "postpay"
+}
```
Comment (rap1ds) [inline app/services/transaction_service/process.md]: This is an array

**Pull request merged**